includes/student-progress/course-progress/repositories/class-course-progress-repository-aggregate.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ *  File containing the Course_Progress_Repository_Aggregate class.
+ *
+ * @package sensei
+ */
+
+namespace Sensei\Student_Progress\Course_Progress\Repositories;
+
+use Sensei\Student_Progress\Course_Progress\Models\Course_Progress_Interface;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class Course_Progress_Repository_Aggregate.
+ *
+ * @since $$next-version$$
+ */
+class Course_Progress_Repository_Aggregate implements Course_Progress_Repository_Interface {
```

Comment:
```suggestion
class Aggregate_Course_Progress_Repository implements Course_Progress_Repository_Interface {
```

Reply:
On second thought, I've decided to remove this class to avoid confusion. More info in [this comment](https://github.com/Automattic/sensei/pull/5428#issuecomment-1234929717).